src/components/scene/keyboard-shortcuts.js

```diff
@@ -0,0 +1,35 @@
+var registerComponent = require('../../core/component').registerComponent;
+var THREE = require('../../lib/three');
```

Comment:
I was trying to make a text editor, and I couldn't input the letter "f" so I really wanted a way to turn that off.